test/Npgsql.Tests/Types/DateTimeInfinityTests.cs

```diff
@@ -13,6 +13,57 @@ namespace Npgsql.Tests.Types;
 #endif
 public class DateTimeInfinityTests : TestBase, IDisposable
 {
+    [Test]
+    public async Task TimestampTz_write_DTO()
```

Comment:
nit: use AssertType and friends instead? I know this test suite doesn't yet, so we can also do the entire thing in a pass later.

Reply:
Will try to see how easy it is to refactor it

Reply:
Thanks for the cleanup, looks much better!